src/shared-components/accordion/index.tsx

```diff
@@ -14,9 +14,20 @@ import {
   Truncate,
 } from './style';
 
-const noop = () => {};
-
-class Accordion extends React.Component {
+type AccordionProps = {
+  children: React.ReactNode;
+  disabled?: boolean;
+  isOpen: boolean;
+  noBorder?: boolean;
+  onClick: Function;
```

Comment:
```suggestion
  onClick: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
```

Reply:
thanks I always forgot how to write these.